helmfile/charts/prometheus-alerts/templates/rules/daily-checks.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.dailyChecks}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "daily-checks" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: daily-checks
+    rules:
+    - alert: S3BucketContentOver{{.Values.s3BucketPercentLimit}}Percent
```

Comment:
don't we want to track also "time since last modified object"?

Reply:
Hmm yeah, maybe! Not for this "alert" in partial maybe but overall indeed. Think such things should be in s3-exporter:

https://github.com/ribbybibby/s3_exporter

Reply:
Did the new rules file based on this idea.